src/common/primitive_hashing.cpp

```diff
@@ -757,6 +768,13 @@ size_t get_desc_hash(const sdpa_desc_t &desc) {
     // Scale type
     seed = hash_combine(seed, static_cast<size_t>(desc.scale_dt));
     seed = hash_combine(seed, desc.invert_scale);
+    seed = hash_combine(seed, desc.kv_head_number);
+    seed = hash_combine(seed, get_runtime_scale_hash(desc.kq_scales));
+    seed = hash_combine(
+            seed, get_zero_points_hash(desc.kq_zero_points, DNNL_ARG_WEIGHTS));
+    seed = hash_combine(seed, get_runtime_scale_hash(desc.vs_scales));
+    seed = hash_combine(
+            seed, get_zero_points_hash(desc.vs_zero_points, DNNL_ARG_WEIGHTS));
```

Comment:
If dynamic quantization makes to the micro-kernel and DNNL_ARG_SRC would hit its usage, it would be not obvious to update this place. Suggest to create one more function that gets a hash for whole zero-points object. Applies to rest places.

Reply:
Nit: please follow the order of members in `sdpa_desc_t`. It's easier to make sure nothing is left behind. APplies to rest places as well.